tools/sizing/size_finder.go

```diff
@@ -0,0 +1,103 @@
+package sizing
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+// next_power_of_2(len(payload)*32/31)
+
+// BlobHeaderSize is the size of the header data on a blob, in bytes.
+const BlobHeaderSize = uint64(31)
+
+// PayloadSizeToBlobSize takes a payload size in bytes and returns the corresponding blob size in bytes.
+// The blob size is the size used for determining payments and throttling by EigenDA. Two payloads of
+// differing length that have the same blob size cost the same and use the same amount of bandwidth.
+func PayloadSizeToBlobSize(payloadSize uint64) uint64 {
+	return encoding.NextPowerOf2(payloadSize*32/31 + BlobHeaderSize)
+}
+
+// BlobSizeToMaxPayloadSize takes a given a blob size and determines the maximum payload size
+// that yields that blob size.
+func BlobSizeToMaxPayloadSize(blobSize uint64) (uint64, error) {
+	if !encoding.IsPowerOf2(blobSize) {
+		return 0, fmt.Errorf("blob size %d is not a power of 2", blobSize)
+	}
+
+	return (blobSize - BlobHeaderSize) * 31 / 32, nil
+}
+
+// BlobSizeToMinPayloadSize takes a given a blob size and determines the minimum payload size
+// that yields that blob size.
+func BlobSizeToMinPayloadSize(blobSize uint64) (uint64, error) {
```

Comment:
Is min payload size actually a value that anyone would care about?

Reply:
I needed it in order to generate my table. In theory I could have derived this from the max size numbers, but I also it makes a nice sanity check on the method that finds the max payload size. We have to compute the number using two completely different pathways, and the unit tests detect if the min/max sizes don't agree with each other.

Reply:
removed column